project1/application.py

```diff
@@ -49,6 +49,12 @@ def register():
     return render_template("register.html")
 
 
+@app.route("/admin")
+def admin():
+    user1=User.query.all()
+    return render_template("userslist.html",name=user1)
```

Comment:
Name your variables properly. Instead of user1 you can name it as 'users'. Keep in mind for future work.

Reply:
sure will do